src/components/Contacts/Contacts.module.css

```diff
@@ -0,0 +1,79 @@
+.wrap {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  width: 500px;
+  padding: 20px;
+  margin: auto;
+}
+
+.title {
+  font-family: 'Courier New', Courier, monospace;
+  margin-bottom: 15px;
+  font-weight: bold;
+  text-align: center;
+  font-size: 35px;
+}
+
+.content {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  font-size: 20px;
+}
+
+.contact {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-start;
+  align-items: center;
+  text-align: left;
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 20px;
+  text-decoration: none;
+  padding: 20px;
+}
+
+.icon {
+  display: block;
+  width: 15%;
+  margin-right: 20px;
+}
+
+.link,
+.phone,
+.location {
+  color: black;
+  font-family: 'Courier New', Courier, monospace;
+  font-size: 20px;
+  text-decoration: none;
+}
+
+.link:hover {
+  text-decoration: underline;
+}
+
+@media (max-width: 780px) {
+  .title {
+    font-size: 20px;
+  }
+
+  .icon {
+    width: 10%;
+    padding: 5px;
+  }
+
+  .link,
+  .phone, 
```

Comment:
Codacy found an issue: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/ReactProject_karolina/pullRequest?prid=6313319)